### ISSUE TITLE
Handwritten Signature: view mode: missing `aria-labelledby`

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#6817]
```html
<span class="xforms-label" id="section-1-section≡grid-1-grid≡signature-control≡≡l">Signature</span>
<span id="section-1-section≡grid-1-grid≡signature-control≡fr-handwritten-signature-elem"
      class="xforms-control xforms-output xforms-mediatype-image xforms-static xforms-readonly xforms-type-anyURI">
    <img
        id="section-1-section≡grid-1-grid≡signature-control≡fr-handwritten-signature-elem≡≡c"
        class="xforms-output-output"
        src="/orbeon-forms-pe/xforms-server/dynamic/ac4297ca39ebac99f28e0ccdce0df3023d74aa59">
</span>
```

**Comment by ebruchez** [comment]
The binding has:

```xml
xxbl:label-for="fr-handwritten-signature-elem"
```

pointing to:

```xml
<xf:output
    id="fr-handwritten-signature-elem"
    mediatype="image/*"
    ref="xxf:binding('fr-handwritten-signature-output')"/>
```
